main/acle.md

```diff
@@ -237,6 +237,12 @@ Armv8.4-A [[ARMARMv84]](#ARMARMv84). Support is added for the Dot Product intrin
 * In [Data types](#data-types), clarified that `__fp16` and `__bf16` are
   predefined types whereas vector types like `int32x4_t` are not.
 * Moved the [Future directions](#future-directions) chapter to the end.
+* Added support for SVE and SVE2. This incorporates the final 00bet6
```

Comment:
Maybe create a link for the text `support for SVE and SVE2` into the main section of the SVE header file`<arm_sve.h>`?